### ISSUE TITLE
clear don't work && vim going crazy

### ISSUE THREAD

**Issue by b4dtR1p** [black-screen/black-screen#39]
the simple clear command don't work, and vim goning crazy!

vim don't recognize direction command and also don't show menu and other plugin installed.

os => osx ElCapitan beta6

**Comment by G07cha** [comment]
This is two separate problems. 
`clear` command investigation result:
- iTerm2/Terminal:
```bash
❯ clear
#Clearing the screen
❯ echo $TERM
xterm-256color
```
- Black Screen:
```bash
$ clear
TERM environment variable not set.
$ echo $TERM
$TERM
```

**Comment by G07cha** [comment]
Same problem as in #36 btw

**Comment by shockone** [comment]
@b4dtR1p, complete Vim support is a long way, we're working on it. 

It's more interesting with clear. It sends an ANSI sequence, which means "move cursor to the upper left corner; erase everything after the cursor". However, in the current design one job (that is when you type `clean` and then press enter) has no access to the output of another job, unlike in xterm. That means that if you types `ls`, the next command can't access its output. 

Currently you can clear the screen with Ctrl+L, but I'll check how we can make clear work as well. I can see two ways:

* Have a built-in `clear` function.
* Redefine the sequence in terminfo.

**Comment by b4dtR1p** [comment]
First of all congratulations for the idea , we lacked a terminal node and thank you for your speedy response!

**Comment by b4dtR1p** [comment]
for environment variable path @G07cha read the TODO #58

**Comment by slang800** [comment]
this issue should be closed since half is covered by #36 and half is very long-term

**Comment by shockone** [comment]
Clear was just added, and vim has been sane for a while.
